### PR TITLE
Add http-to-jms protocol switch template

### PR DESCRIPTION
**Pull request by ThishaniLucas**

### CODE REVIEW

**Comment by ThishaniLucas** [comment]
Need to merge only after the balo-cache location resolving issue is resolved.

File: templates/protocol_switch_service/Ballerina.toml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+org-name= "wso2"
+version= "1.0.0"
+license= ["Apache-2.0"]
+authors= ["WSO2"]
+keywords= ["ballerina", "integrator"]
+repository= "https://github.com/wso2/ballerina-integrator"
+templates= ["protocol_switch_service"]
+
+[dependencies]
+"wso2/jms"= {path = "/home/thishani/ballerina-modules/module-jms/target/balo/jms-2019r3-java8-0.6.0.balo"}
```
Comment (IrushiL) [inline templates/protocol_switch_service/Ballerina.toml]: Can we give a relative path here? Or we could let it be used as a module pulled from Ballerina Central without pointing the balo?

File: templates/protocol_switch_service/Ballerina.toml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  module = "protocol_switch_service"
+  path = "./lib/activemq-client-5.15.5.jar"
+
+  [[platform.libraries]]
+  module = "protocol_switch_service"
+  path = "./lib/geronimo-j2ee-management_1.1_spec-1.0.1.jar"
+
+  [[platform.libraries]]
+  module = "protocol_switch_service"
+  path = "./lib/hawtbuf-1.11.jar"
```
Comment (IrushiL) [inline templates/protocol_switch_service/Ballerina.toml]: New line

File: templates/protocol_switch_service/src/protocol_switch_service/Module.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+3. Add the following jars from ACTIVE_MQ_HOME/lib to <TEMPLATE_HOME>/lib folder.
+
+* activemq-client-{activemq-version}.jar
+* geronimo-j2ee-management_1.1_spec-1.0.1.jar
+* hawtbuf-1.11.jar"
+
+
+4. Execute the following command in the project directory scatter_gather_service to run the service.
+```bash
+ballerina run --config src/protocol_switch_service/resources/ballerina.conf protocol_switch_service
```
Comment (IrushiL) [inline templates/protocol_switch_service/src/protocol_switch_service/Module.md]: --config is now changed to "--b7a.config.file=path/to/file"

File: templates/protocol_switch_service/src/protocol_switch_service/resources/ballerina.conf [hunk 4]
```diff
@@ -0,0 +1,3 @@
+LISTENER_PORT = 9090
+INITIAL_CONTEXT_FACTORY = "org.apache.activemq.jndi.ActiveMQInitialContextFactory"
+PROVIDER_URL = "tcp://localhost:61616"
```
Comment (IrushiL) [inline templates/protocol_switch_service/src/protocol_switch_service/resources/ballerina.conf]: New line

File: templates/protocol_switch_service/Ballerina.toml (hunk 1 above)
Comment (ThishaniLucas) [reply templates/protocol_switch_service/Ballerina.toml]: No need of this, hence removed.

File: templates/protocol_switch_service/Ballerina.toml (hunk 2 above)
Comment (ThishaniLucas) [reply templates/protocol_switch_service/Ballerina.toml]: fixed

File: templates/protocol_switch_service/src/protocol_switch_service/Module.md (hunk 3 above)
Comment (ThishaniLucas) [reply templates/protocol_switch_service/src/protocol_switch_service/Module.md]: fixed

File: templates/protocol_switch_service/src/protocol_switch_service/resources/ballerina.conf (hunk 4 above)
Comment (ThishaniLucas) [reply templates/protocol_switch_service/src/protocol_switch_service/resources/ballerina.conf]: fixed

**Pull request merged**